vm/src/stdlib/zlib.rs

```diff
@@ -246,7 +246,10 @@ mod decl {
                 if stream_end {
                     Ok(buf)
                 } else {
-                    Err(new_zlib_error("incomplete or truncated stream", vm))
+                    Err(new_zlib_error(
+                        "Error -5 while decompressing data: incomplete or truncated stream",
+                        vm,
+                    ))
```

Comment:
@coolreader18 @youknowone In the future, could we look at dynamically generating these error messages in `new_zlib_error`, like [how `zlib_error` is implemented in the CPython source](https://github.com/python/cpython/blob/3.8/Modules/zlibmodule.c)?

Reply:
Hmm, that'd be nice, but flate2 doesn't expose any information about the specific error that occurred, so we'd need to change it there first.

Reply:
It seems like [flate2's `Status` enum](https://docs.rs/flate2/1.0.20/flate2/enum.Status.html) could be expanded to account for more than just `BufError`, but I could be wrong.

Reply:
I think it should be possible to fetch zlib's `msg` in flate2, it should give better error messages there as well. I'll look into contributing that.